git_your_code_on/jdirico_traffic-light/traffic-light.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="css/main.css">
+	<title>Traffic Light</title>
+</head>
+
+<body>
+<div style="float: left">
+<ul class="buttons">
+	<li class="stop">Stop</li>
+	<li class="slow">Slow</li>
+	<li class="go">Go</li>
+</ul>
+</div>
+<div style="float: left; background-color: #2B2B2B; margin-top: 40px; margin-bottom: 40px; border-radius: 3em;" >
```

Comment:
Also the inline css is not best practice (as discussed in class) so you may want to pull that out into your css file